massa-models/src/config/constants.rs

```diff
@@ -84,6 +84,21 @@ lazy_static::lazy_static! {
         .parse()
         .unwrap()
     };
+    /// node chain id (to avoid replay attacks)
+    pub static ref CHAINID: u64 = {
+        // MASSA..
+        // MASM (MainNet):           77658377
+        // MASB (BuildNet):          77658366
+        // MASS (SecureNet):         77658383
+        // MASS275 (SecureNet 27.5): 77658383505553
+        // MASL (Labnet):            77658376
+        // SANDBOX (Sandbox):        77
+        if cfg!(feature = "sandbox") {
+            77
+        } else {
+            77658383
+        }
+    };
```

Comment:
Can you make an issue in private-deployment to add the chain id config for the various networks we deploy?

Reply:
private-deploymenent PR aready there :)